pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "supervision"
+version = "0.12.0"
+description = "A set of easy-to-use utils that will come in handy in any Computer Vision project"
+authors = ["Piotr Skalski <piotr.skalski92@gmail.com>"]
+maintainers = ["Piotr Skalski <piotr.skalski92@gmail.com>"]
+readme = "README.md"
+packages = [{include = "supervision"}]
+homepage = "https://github.com/roboflow/supervision"
+repository = "https://github.com/roboflow/supervision"
+documentation = "https://github.com/roboflow/supervision/blob/main/README.md"
+keywords = ["machine-learning", "deep-learning", "vision", "ML", "DL", "AI", "YOLOv5", "YOLOv8", "Roboflow"]
+
+classifiers=[
+    'Intended Audience :: Developers',
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: BSD License',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3 :: Only',
+    'Topic :: Software Development',
+    'Topic :: Scientific/Engineering',
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: Image Recognition",
+    'Typing :: Typed',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: Unix',
+    'Operating System :: MacOS'
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.12.0"
+numpy = "^1.20.0"
+opencv-python-headless = "^4.8.0.74"
+matplotlib = "^3.7.1"
+pyyaml = "^6.0"
+opencv-python= { version = "^4.8.0.74", optional = true }
```

Comment:
Is it possible to make `opencv-python-headless` `optional` and `opencv-python` default? And if so, how would that work?

I'm asking because most of our users are probably `desktop` users, but there is a growing need for `headless` installation. I want to support both, but I still prioritize `desktop`.

Reply:
I sent the proper commit and my comments about it.

Reply:
In that case, let's bring back `opencv-python` as not `optional`. We need to install some version of `opencv` by default.